tools/ubergen/bin/ubergen.ts

```diff
@@ -260,7 +268,30 @@ async function transformPackage(
 ) {
   await fs.mkdirp(destination);
 
-  await copyOrTransformFiles(library.root, destination, allLibraries, uberPackageJson);
+  if (uberPackageJson.ubergen?.stripExperimental && library.packageJson.stability === 'experimental') {
+    // in stripExperimental is on want to only add the L1s of experimental modules.
+    let cfnScopes = library.packageJson['cdk-build'].cloudformation;
+
+    if (cfnScopes === undefined) {
+      return false;
+    }
+
+    if (typeof cfnScopes === 'string') {
+      cfnScopes = [cfnScopes];
+    }
+    const destinationLib = path.join(destination, 'lib');
+    await fs.mkdirp(destinationLib);
+    await cfn2ts(cfnScopes, destinationLib);
+    // create a lib/index.ts which only exports the generated files
+    fs.writeFileSync(path.join(destinationLib, 'index.ts'),
+      /// logic copied from `create-missing-libraries.ts`
+      cfnScopes.map(s => (s === 'AWS::Serverless' ? 'AWS::SAM' : s).split('::')[1].toLocaleLowerCase())
+        .map(s => `export * from './${s}.generated';`)
+        .join('\n'));
```

Comment:
minor: maybe we can move this logic into cfn2ts, and remove it from cfnspec and here?

Reply:
> `cfn2ts` is intended to be a general tool to generate ts class from the CFN spec.

That's fine. I'm not going to fight it.

But I will point out that this same argument can be made about `ubergen`, i.e., a general tool to take a set of node modules and produce an 'uber' package. 
For the same reason, I had suggested (in our call) to move this logic out of ubergen and into a script specific to `aws-cdk-lib`.

Reply:
Moving it to `aws-cdk-lib` means that we will have to duplicate the logic for transforming files (the rewrites), which is non trivial. `ubergen` is not a general tool, it is tailored to the AWS CDK repo. It reference `@aws-cdk` as the scope of the packages to bundle, has special treatment for `cfnInclude` and has special case for the `core` module.

Reply:
Ok.